admin/templates/home.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
```

Comment:
webpack_loader isn't in requirements.txt or requirements/dev.txt. I installed it by hand and things started working but the server wouldn't run until I did because it's in installed apps.